src/views/forms/AccountManagementForms/DepositForm.tsx

```diff
@@ -176,7 +177,7 @@ export const DepositForm = ({ onDeposit, onError }: DepositFormProps) => {
 
   // Async Data
   const { balance } = useAccountBalance({
-    addressOrDenom: sourceToken?.address || CHAIN_DEFAULT_TOKEN_ADDRESS,
+    addressOrDenom: sourceToken?.address || NATIVE_TOKEN_ADDRESS,
```

Comment:
this should really be `??` and i *can* change it, but we're going to get rid of this soon anyway... 👀

Reply:
what do u mean by getting rid of this? 👀

Reply:
the `DepositForm` component 👀 since we're rewriting it